### PR TITLE
[transformation] Comments removal transformation

### PR DESCRIPTION
**Pull request by nbirillo**

### CODE REVIEW

**Comment by artbobrov** [comment]
Does it work for multiline string?
For example:
```python
tmp = '''
hello
world
'''
```

**Comment by nbirillo** [comment]
@artbobrov tnaks! I will fix it

File: src/main/kotlin/org/jetbrains/research/ml/ast/transformations/commentsRemoval/CommentsRemovalTransformation.kt [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    }
+
+    override fun apply(psiTree: PsiElement, toStoreMetadata: Boolean) {
+        val comments = PsiTreeUtil.collectElementsOfType(psiTree, PsiComment::class.java)
+        val stringLiteralExpressions = PsiTreeUtil.collectElementsOfType(psiTree, PyStringLiteralExpression::class.java)
+
+        val visitor = CommentsRemovalVisitor()
+
+        acceptStatements(psiTree.project, comments, visitor)
+        acceptStatements(psiTree.project, stringLiteralExpressions, visitor)
```
Comment (elena-lyulina) [inline src/main/kotlin/org/jetbrains/research/ml/ast/transformations/commentsRemoval/CommentsRemovalTransformation.kt:25]: acceptStatements(psiTree.project, comments + stringLiteralExpressions, visitor) ?

**Review by elena-lyulina — CHANGES_REQUESTED** [review]

**Review by elena-lyulina — APPROVED** [review]
Cool

**Pull request merged**